app/index.html

```diff
@@ -13,7 +14,27 @@ <h1>HELLO WORLD!</h1><i class="fa fa-hand-spock-o" aria-hidden="true"></i>
   	<div class="header header__second"></div>
   	<div class="header header__third"></div>
   	<div class="header header__fourth"></div>
-  	<div class="header header__fifth"></div>
+  	<div class="header header__fifth">
+			<main class="fifth">
+				<nav class="fifth__menu">
+					<img alt="Logo UI" src="img/logo.png" class="fifth__menu--logo">
+					<ul class="fifth__navigation">
+						<a class="fifth__navigation--item"><li>PRODUCTS</li></a>
```

Comment:
when you make a list, it should be:
```
<ul>
  <li>
    <a></a>
  </li>
  <li>
    <a></a>
  </li>
  <li>
    <a></a>
  </li>
</ul>

```
And add please required `href` attribute to all the `a` tags

Reply:
Corrected, but please explain why we need nest 'a' in 'li', but not in reverse.

Reply:
because `a` tag is link. `li` tag is item of the list. So if you are making list of links, as now, you should put `a` tags inside `li`. 
Every time you want to make list of something you should follow rule, that all the items of list(for example links or products, or options, etc) are inside `li` tag.

Reply:
Okey, but if I have order list and I need link when I click on number too?
In that example I can't do it, only click on the context inside `li`.
Is this an exception or how?